Sources/Extensions/Widgets/Gauge/WidgetGauge.swift

```diff
@@ -0,0 +1,36 @@
+import Intents
+import Shared
+import SwiftUI
+import WidgetKit
+
+@available(iOS 17, *)
+struct WidgetGauge: Widget {
+    var body: some WidgetConfiguration {
+        AppIntentConfiguration(
+            kind: "io.robbie.HomeAssistant.widget-gauge",
```

Comment:
Please use an identifier such as what we use in `WidgetActionsIntent.widgetKind'

Just a heads up that the app bundle is dynamic and not always `io.robbie.HomeAssistant`